frontend/src/views/activity/Category.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-container fluid>
+    <content-card toolbar>
+      <template #title>
+        <v-toolbar-title class="font-weight-bold">
+          <template v-if="category()">
+            <v-chip
+              :color="category().color"
+              dark>
+              {{ category().short }}
+            </v-chip>
+            {{ category().name }}
+          </template>
+          <template v-else>
+            loading...
+          </template>
+        </v-toolbar-title>
+      </template>
+      <template #title-actions>
+        <v-btn v-if="!layoutMode"
```

Comment:
Diese 2 Buttons könnte man in eine separate Komponente auslagern. Ist jetzt einfach copy & past von Activity, oder?

Reply:
Können wir machen.
Langfristig glaube ich eher, dass wir die Buttons zusätzlich anders beschriften werden:
- Layout anpassen (default)
- Standard-Inhalte definieren
Da stellt sich dann die Frage, ob es sich lohnt, das über eine einheitliche Komponente zu lösen.

Reply:
Ok, d'accord